docs/installation.md

```diff
@@ -1,101 +1,44 @@
 # Installing Kusk Gateway
 
----
-**NOTE**
-
-This documents describes the installation of the Kusk Gateway and its load balancing component Envoy Fleet to the generic Kubernetes cluster.
-
-If you're looking for the quick way to try Kusk Gateway in a locally setup Kubernetes cluster, please see [Local Installation with Minikube](local-installation.md).
```

Comment:
I suggest dropping local-installation.md in favor of the small note here https://github.com/kubeshop/kusk-gateway/pull/285/files#diff-8b042e3f94ca5c59a7cd990b950aec0073ea84fe811e7b22be51158d7b180d56R40

As I believe it complicates too much and it could be solved simply with `minikube service`

Reply:
sounds good!

Reply:
Does this mean we no longer have a need for the `development/cluster/create-cluster.sh` script then?

Reply:
Would still work for bare-metal.. WDYT, should we keep it or?

Reply:
the create cluster script actually creates a minikube cluster profile and sets up the metallb config. These steps would need to be removed to make the script "cluster provider agnostic"